components/popupWithImage.js

```
@@ -0,0 +1,29 @@
+import Popup from './popup.js'
+
+export default class PopupWithImage extends Popup {
+    constructor(popupSelector) {
+        super(popupSelector)
+    }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)